ts/parts/Stacking.ts

```diff
@@ -652,7 +656,12 @@ Axis.prototype.cleanStacks = function (this: Highcharts.Axis): void {
 };
 
 
-// Stacking methods defnied for Series prototype
+// Stacking methods defined for Series prototype
+Series.prototype.setGroupedPoints = function (this: Highcharts.Series): void {
+    if (this.options.centerInCategory) {
+        Series.prototype.setStackedPoints.call(this, 'group');
```

Comment:
Hmm.. why not simply `this.setStackedPoints('group')`?

Reply:
Because many of the box-like series types override `setStackedPoints`. Haven't tested it though.

Reply:
Ah, that's the case! Thanks 👍